prog/dftb+/lib_io/ipisocket.F90

```diff
@@ -233,7 +233,7 @@ subroutine receive(this, coord, cell, tStop)
 
     ! lattice vector data
     call readbuffer(this%socket, commsBuffer2)
-    cell(:,:) = reshape(commsBuffer2, [3, 3])
+    cell(:,:) = transpose(reshape(commsBuffer2, [3, 3])) ! eae32
```

Comment:
What does the comment denote? As this is versioned, the author information is available in the log.

Reply:
When I was going through the very painful debugging process, Edgar Engel gave some help. 

Okay I'll remove the comment.